modules/api/src/main/java/com/intuit/wasabi/api/AssignmentsResource.java

```diff
@@ -645,11 +622,94 @@ public Response clearMetadataCache(@HeaderParam(AUTHORIZATION) @ApiParam(value =
     @Produces(APPLICATION_JSON)
     @ApiOperation(value = "Get assignments metadata cache details...")
     @Timed
-    public Response getMetadataCacheDetails(@HeaderParam(AUTHORIZATION) @ApiParam(value = EXAMPLE_AUTHORIZATION_HEADER, required = true) final String authorizationHeader) {
-        UserInfo.Username userName = authorization.getUser(authorizationHeader);
-        authorization.checkSuperAdmin(userName);
-
+    public Response getMetadataCacheDetails() {
         return httpHeader.headers().entity(assignments.metadataCacheDetails()).build();
     }
 
+    /**
+     * Convert Assignment object collection to the List of response MAP expected by the end user.
+     *
+     * @param assignments
+     * @return
+     */
+    protected List<Map<String, Object>> toBatchAssignmentResponseMap(Collection<Assignment> assignments) {
```

Comment:
Cosmetic: Suggest changing method name to getBatchAssignmentResponseMapsfromAssignmentCollection() to better communicate intent.